### ISSUE TITLE
Compare unique items not system outputs

### ISSUE THREAD

**Issue by mjpost** [cfedermann/Appraise#45]
Many times, the systems outputs for a sentence are identical. Rather than constructing each task from a random subset of *systems*, each task should be constructed from the set of distinct *outputs* for that sentence. The pairwise rankings could then be re-associated with the systems to generate a larger set of pairwise rankings.

This would be a bit more respectful of people's times (it's annoying to see identical outputs), and would also let us potentially gather data more quickly. On the WMT14 data, for example, there are identical system outputs on over half the sentences.

CC: @cfedermann

**Comment by cfedermann** [comment]
This will be fixed for WMT15

**Comment by mjpost** [comment]
I found a workaround for this, that creates entries where the system name is a comma-delimited list of systems. You then just have to split those out and compile out the (often much larger) set of rankings. If you want what I've done, let me know. That might be a better way than trying to do it internally.

**Comment by cfedermann** [comment]
@mjpost elegant solution; I didn't plan to do this deduping internally.  Data will be rendered as-is.

Can you point me to your code for this?

**Comment by cfedermann** [comment]
Commit 00131810b5 addresses this during batch generation...

**Comment by cfedermann** [comment]
@mjpost I'm preparing sample files next;  it would be nice if you could have a quick look when you get a chance...

**Comment by mjpost** [comment]
Yes, please send them. I likely won't have time till later in the day but will prioritize it.

**Comment by cfedermann** [comment]
Aloha @mjpost, mini batches are inside the repo (new wmt15data folder). They look good to me but I'm a little worn out by now ;)

Any feedback you might have is very welcome.

Cheers and best,
  Christian



-----Original Message-----

**Comment by cfedermann** [comment]
@mjpost Batch 1 files have been added (wmt15data/full-batches folder).

I have checked that exporting data for "multi systems" generates the right CSV format, possibly spanning more than a single line. I add one or more  PLACEHOLDER systems to make sure we end up with five systems per row. The corresponding rank is -1, so this does not have an effect on scoring. Will verify that soon...

Let me know if you spot any issues with the data.

**Comment by mjpost** [comment]
Thanks. Kann ich einen invite token?
